en/SUMMARY.md

```diff
@@ -45,10 +45,18 @@
   * [Battery](config/battery.md)
   * [Safety](config/safety.md)
     * [Failsafe Simulation](config/safety_simulation.md)
+  * [ESC Calibration](advanced_config/esc_calibration.md)
```

Comment:
FYI @MaEtUgR Note this moved here under standard config, because it is useful in all cases. This really should be in its own section in QGC.

Reply:
I don't think it should be in QGC. It's quite autopilot-specific and if I were able to adjust QGC accordingly it would get much easier for the user...

Reply:
> I don't think it should be in the QGC docs. It's quite autopilot-specific and if I were able to adjust QGC accordingly it would get much easier for the user...

@MaEtUgR I'm not suggesting it should be in the QGC _docs_ - though I guess I did not provide enough context sorry. 

I'm suggesting that ESC calibration should not be part of Batter/Power configuration in QGC but instead its own specific view in QGC. It might be even be better as part of actuator config, though that interface is already quite "full"

For this docs change I mostly wanted you to be aware that this is now part of the "standard" instructions.